tests/src/test/java/org/wso2/micro/gateway/tests/extensions/OASAPIInvokeTestCase.java

```diff
@@ -67,4 +68,9 @@ public void testApiInvokeWithJWT() throws Exception {
         Assert.assertEquals(response.getData(), ResponseConstants.petByIdResponse);
         Assert.assertEquals(response.getResponseCode(), 200, "Response code mismatched");
     }
+
+    @AfterClass
+    public void stop() throws Exception {
+        super.finalize();
+    }
```

Comment:
This test class no need to have after class. This will cause test failures

Reply:
fixed